dspace/config/emails/subscription

```diff
@@ -0,0 +1,12 @@
+# E-mail sent to DSpace users when new items appear in collections they are
+# subscribed to
+#
+# Parameters: {0} is the details of the new collections and items
+# See org.dspace.core.Email for information on the format of this file.
+#
+Subject: DSpace Subscription
+New items are available in the collections you have subscribed to:
+
+{0}
+
+DSpace
```

Comment:
Would you want this to be "The DSpace Team" instead or is this intentionally just "DSpace"?

Reply:
Email was ported over from TAMU DSpace6. 

@wwtamu -- Do we want to change this do "The DSpace Team" to be inline with the other templates?

Reply:
The upstream dspace6x branch seems to have "DSpace" rather than "The DSpace Team":
- https://github.com/TAMULib/DSpace/blob/dspace-6_x/dspace/config/emails/subscription